ansible/group_vars/test

```diff
@@ -1,3 +1,5 @@
 jenkins_url: "https://ci.nodejs.org"
 jenkins_worker_jar: "{{ jenkins_url }}/jnlpJars/slave.jar"
 server_user: iojs
+home: "/users"
```

Comment:
Can this be moved to the host_vars template? Also from `release`.

Reply:
If I move it out then every time somebody creates as host_vars for other platforms they would have to add that (we currently only have a template for s390x).  It is the default to be used in cases other than z/OS, and its overridden in the ansible.cfg for the z/OS platform.  

I think its better to keep it here than having people need to remember to add it when creating their host_vars files...

Reply:
I was reading this wrong before, in that case this is the correct palce. But shouldn't this be `/home` instead?

If I'm reading this right, your objective is to replace references to `/home` with `{{ home }}`, which sounds good to me. But the default should be what was there before replacing, that is `/home` not `/users`. At least the `download slave.jar` step will fail on all other platforms with this line (unless I'm still missing something).

Reply:
Thanks for catching that, I'm not sure where I came up with users as it should be home.  Pushed commit to fix.